sn_networking/src/event/request_response.rs

```diff
@@ -242,8 +244,17 @@ impl SwarmDriver {
 
         let event_sender = self.event_sender.clone();
         let _handle = tokio::spawn(async move {
-            let keys_to_verify =
-                Self::select_verification_data_candidates(&peers, &all_keys, &sender);
+            let keys_to_verify: Vec<_> = {
+                let mut vec = Self::select_verification_data_candidates(&peers, &all_keys, &sender)
```

Comment:
The `select_verification_data_candidates` function itself only return 1 chunk candidate to be verified.
Hence, this PR has no effect on reducing the workload.